src/components/NavSection.vue

```diff
@@ -34,6 +34,9 @@
         <a class="navbar-item" href="/collections">Collections</a>
         <a class="navbar-item is-hidden-desktop" href="/search-help">Search Guide</a>
         <a class="navbar-item" href="/feedback">Feedback</a>
+        <a class="navbar-item" href="https://opensource.creativecommons.org/ccsearch-browser-extension/" target="_blank">
+        <img src="../assets/external-link.svg" />
```

Comment:
You should use the `external-link` icon. See docs here https://cc-fonts.netlify.app/?path=/docs/vocabulary-icons--page

Reply:
Done :)